### PR TITLE
Optimizely Audience Doc

### PR DESCRIPTION
**Pull request by Romani-Ramzi**
### Proposed changes

Create documentation for Optimizely Audience sync

### Merge timing
 -> ASAP once approved

### CODE REVIEW

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+**Within your Optimizely Data Platform Account**
+1. Navigate to the **App Directory** in the top right (small boxes with + sign icon)
+2. Use the autocomplete to find the **Twilio Segment** App
+3. Click into the app and click the blue **INSTALL** button in the top left
+4. Click the **SETTINGS** tab -> Click **GENERATE** -> and copy the resulting token to your clipboard
+
+**Within your Twilio Segment Account**
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:24]:
```suggestion
2. Click on the Destinations Tab and search for **Optimizely Advanced Audience Targeting**.
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+3. Click **Configure Optimizely Advanced Audience Targeting**.
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:25]:
```suggestion
3. Click on the **Configure Optimizely Advanced Audience Targeting** tile.
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+4. Select your **Personas** source as the source (The app will not work with any other source)
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:26]:
```suggestion
4. Select your **Engage Space** as a Source. Note that this Destination will only work when an Engage Space is configured as a Source.
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+5. Click **Confirm Source**
+6. Within the **Settings** tab paste your ODP token into the **API Key** field, select your region, enable the integration and click **Save**
+7. Click into the **Mappings** section and click **New Mapping**
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:29]:
```suggestion
7. Click into the **Mappings** section, then click **New Mapping**, then click on the **Sync Audience** tile. 
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+8. In section 3 **Select Mappings** ensure the user identifier you are targeting with your Advanced Audience Targeting integration is mapped to the Optimizely User ID field.
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:30]:
```suggestion
8. In section 3 **Select Mappings** ensure the user identifier you are targeting with your Advanced Audience Targeting integration is mapped to the Optimizely User ID field.
9. Click the Save button to save the Action. 
10. Enable the Action by toggling the Status to **Enabled**.
11. Click back to the **Settings** tab and enable the Destination by toggling the **Enable Destination** toggle to "On", then click "Save Changes". 
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+4. Click the **SETTINGS** tab -> Click **GENERATE** -> and copy the resulting token to your clipboard
+
+**Within your Twilio Segment Account**
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure Optimizely Advanced Audience Targeting**.
+4. Select your **Personas** source as the source (The app will not work with any other source)
+5. Click **Confirm Source**
+6. Within the **Settings** tab paste your ODP token into the **API Key** field, select your region, enable the integration and click **Save**
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:28]:
```suggestion
6. Within the **Settings** tab paste your ODP token into the **API Key** field, select your region, enable the integration and click **Save Changes**
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+7. Click into the **Mappings** section and click **New Mapping**
+8. In section 3 **Select Mappings** ensure the user identifier you are targeting with your Advanced Audience Targeting integration is mapped to the Optimizely User ID field.
+
+{% include components/actions-fields.html %}
+
+### **Notes for Using the Destination**
+
+- Ensure the Advanced Audience Targeting integration is configured in your Optimizely Products so that you can access your connected audiences from Twilio Segment in your Optimizely Product (Web Experimentation, Feature Experimentation, and CMS Visitor Groups). Documentation:
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:36]: Can this be reworded slightly? Maybe break up the sentence?

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+[Web Experimentation Advanced Audience Targeting](https://docs.developers.optimizely.com/web-experimentation/docs/configure-odp-audience-targeting#integrate-with-other-cdps)
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:38]: Can we explain why we are linking to this location please?

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+[Feature Experimentation Advanced Audience Targeting](https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting)
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:40]: Can we explain why we are linking to this location please?

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+[CMS Visitor Groups Advanced Audience Targeting](https://github.com/unrvld/ODP.VisitorGroups)
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:42]: Can we explain why we are linking to this location please?

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+- If connecting audiences to **Optimizely Web Experimentation** ensure the user id you map to the Optimizely User ID in the destination mappings area is an identifier available on the browser (client side)
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:44]: Can we make this sentence a little clearer to understand please

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
+---
+Optimizely Advanced Audience Targeting Destination
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[Optimizely Advanced Audience Targeting](https://optimizely.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) allows you to sync your Segment Engage audiences to Optimizely for targeting with Web Experimentation audiences, Feature Experimentation audiences, and CMS Visitor Groups.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:7]:
```suggestion
[Optimizely Advanced Audience Targeting](https://optimizely.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} allows you to sync your Segment Engage audiences to Optimizely for targeting with Web Experimentation audiences, Feature Experimentation audiences, and CMS Visitor Groups.
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 8 above)
Comment (tcgilbert) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:38]:
```suggestion
[Web Experimentation Advanced Audience Targeting](https://docs.developers.optimizely.com/web-experimentation/docs/configure-odp-audience-targeting#integrate-with-other-cdps){:target="_blank"}
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 10 above)
Comment (tcgilbert) [inline src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md:42]:
```suggestion
[CMS Visitor Groups Advanced Audience Targeting](https://github.com/unrvld/ODP.VisitorGroups){:target="_blank"}
```

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 1 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 2 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 3 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 4 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 11 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 10 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md [hunk 13]
```diff
@@ -0,0 +1,44 @@
+---
+Optimizely Advanced Audience Targeting Destination
+---
```
[the remark this answers is not in the document]
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 8 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 10 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 9 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 8 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 7 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

File: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md (hunk 6 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md]: Fixed

**Comment by stayseesong** [comment]
I'm closing this PR out as I created an editable version for the docs team here: #5512

**Pull request closed without merging**